spec/locales.js

```diff
@@ -20,6 +20,7 @@ const LOCALES = [
   'th',
   'zh-cn',
   'zh-tw',
+  'uk',
```

Comment:
Please insert `uk` section in alphabetic order;
in this case after `th` section.